### ISSUE TITLE
[web] disabling / muting breakpoints has no effect

### ISSUE THREAD

**Issue by johnpryan** [flutter/flutter#73673]
The IntelliJ debugger stops at breakpoints regardless of whether they are removed or "Mute breakpoints" is enabled.

Steps to reproduce:

1. set breakpoint
2. Run app in IntelliJ with Chrome device selected
2. Enable "Mute breakpoints" in IntelliJ, but debugger stops at breakpoint anyway
4. Remove breakpoint in editor, debugger still stops at the breakpoint

**Comment by johnpryan** [comment]
Can't reproduce

**Comment by johnpryan** [comment]
Reopening, this is happening for me on the dev channel.

**Comment by pedromassangocode** [comment]
I was able to reproduce this on latest master channel.

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel master, 1.26.0-13.0.pre.178, on Mac OS X 10.15.7 19H114 darwin-x64, locale en)
    • Flutter version 1.26.0-13.0.pre.178 at /Users/pedromassango/Code/flutter_master
    • Framework revision 22f1700427 (17 minutes ago), 2021-01-26 14:34:03 +0800
    • Engine revision 6e391c5942
    • Dart version 2.12.0 (build 2.12.0-257.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/pedromassango/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    ! CocoaPods 1.9.3 out of date (1.10.0 is recommended).
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To upgrade see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] IntelliJ IDEA Community Edition (version 2020.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 52.2.5
    • Dart plugin version 203.6912

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • Mac OS X 10.15.7 19H114 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 88.0.4324.96

! Doctor found issues in 1 category.
```

</details>

**Comment by osaxma** [comment]
Same happens with VS Code (master 1.27) -- disabling/removing a breakpoint in Flutter Web has no effect. 

<details><summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel master, 1.27.0-2.0.pre.102, on Mac OS X 10.15.6 19G2021 darwin-x64, locale en)
    • Flutter version 1.27.0-2.0.pre.102 at /Users/osaxma/fvm/versions/master
    • Framework revision b2e05062b6 (2 days ago), 2021-02-15 03:56:03 -0500
    • Engine revision 1d537824d6
    • Dart version 2.13.0 (build 2.13.0-30.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/osaxma/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 88.0.4324.150

• No issues found!
```

<details>

**Comment by jaredbaszler** [comment]
This seems to have been fixed for me in the latest Flutter 2.2 (and Dart 2.13) release.  I've tried it a few times today and removing breakpoints no longer pauses it where the breakpoint used to be.  Call me happy as this was kind of a pain until now.

**Comment by johnpryan** [comment]
Thanks @jaredbaszler, yep this is working now.
